Ingest MSCL cleanup

Hwesta opened the pull request:
Numerous small fixes. Notable ones include:
- Merging the Approve Normalization nodes. ProcessingMCP.xml now works to automate Approve Normalization in all cases
- DIP creation is based on the existence of a DIP directory, which is created in all access normalization paths
- Cleanup of microservice groups in Normalization. Added Process manually normalized files, Generate AIP METS and Verify checksums, and ensure that microserviceGroups are contiguous
- Removed dead DIPfromAIP MSCL & scripts
- Delete all TasksConfigs and StandardTasksConfigs with no foreign keys pointing at them.

Recommend checking this out and reviewing the graphlinks and normalization behaviours.

Currently includes #186 but will be merged after that.

Hwesta committed:
MSCL: Ingest cleanup

Many SQL updates:
* Fix typos, remove bizarre links
* Update microservice groups to make them more contiguous
* Verify checksums moved to after metadata processing, as metadata
generates checksums too
* Metadata reminder moved to before submission documentation and metadata
processed
* Remove old style file ID
* Update reingest chain to use existing DIP and AIP creation, instead of
custom path
* Remove useless magic link, clean up setup for normalization and setting
which normalization node to go to
* Remove assignFauxFileUUIDs and renameDIPFauxToOrigUUIDs as we won't
be using fake IDs to separate the original SIP from the reingested AIP
* Delete all TasksConfigs and StandardTasksConfigs that have no foreign
keys pointing at them

Hwesta committed:
MSCL: DIP creation based off of DIP directory, not unitVars

refs #6868

Approve normalization had two MicroServiceChainLinks nodes, which
caused problems since the new processingMCP matches based on UUID not
node name.  However, the only reason to have two approve nodes was to
set whether or not a DIP was created.

Base DIP creation on the existence of a DIP directory created by the access
normalization paths. Merge the Approve Normalization nodes, and remove
unitVars that previously controlled path.

Hwesta committed:
MSCL: Remove duplicate normalize for preservation chain

Two chains existed for preservation normalization. Remove duplicate, point
at other chain.

mistydemeo commented on src/MCPClient/lib/clientScripts/manualNormalizationCheckForManualNormalizationDirectory.py:
```diff
@@ -20,31 +20,25 @@
+    mn_preserve_path = os.path.join(manualNormalizationPath, "preservation")
+    # Return to indicate manually normalized files exist
+    if os.path.isdir(mn_access_path) and os.listdir(mn_access_path):
+        print('Manually normalized files found')
```
Maybe specify which files were found when returning?

Hwesta replied:
Thought about it and decided not to. Partly because it could be a very long list, and because this returns true if access _or_ preservation copies found - if both were found only one list would be printed because we exit immediately (or doing the other check just so we could print the other set of files)

mistydemeo commented:
Need to test, but looks good. :+1:

Hwesta commented on src/MCPServer/share/mysql_dev_6488_aip_reingest.sql:
```diff
@@ -1,3 +1,125 @@
+UPDATE MicroServiceChainLinks SET defaultNextChainLink=@verifychecksumMSCL WHERE defaultNextChainLink='54b73077-a062-41cc-882c-4df1eba447d9';
+UPDATE MicroServiceChainLinksExitCodes SET nextMicroServiceChainLink=@verifychecksumMSCL WHERE nextMicroServiceChainLink='54b73077-a062-41cc-882c-4df1eba447d9';
+--   verify point to new successor
+UPDATE MicroServiceChainLinks SET defaultNextChainLink='54b73077-a062-41cc-882c-4df1eba447d9' WHERE pk=@verifychecksumMSCL;
```
This should be removed to not revert #190

Hwesta committed:
sharedDirectory: Clean up .gitignore structure

Add for all directories, update .gitignore to ignore everything in those
directories.
